.github/workflows/ci.yml

```diff
@@ -183,3 +183,49 @@ jobs:
       run: make clippy-full
     - name: Lint (release)
       run: make clippy-release
+  minimal_versions:
+    name: Minimal versions
+    strategy:
+      matrix:
+        build: [ linux, windows, mac, minimal, default ]
+        include:
+          - build: linux
+            os: ubuntu-latest
+            rust: "stable"
+            features: "full"
+          - build: windows
+            os: windows-latest
+            rust: "stable"
+            features: "full"
+          - build: mac
+            os: macos-latest
+            rust: "stable"
+            features: "full"
+          - build: minimal
+            os: ubuntu-latest
+            rust: "stable"
+            features: "minimal"
+          - build: default
+            os: ubuntu-latest
+            rust: "stable"
+            features: "default"
```

Comment:
Like with escargot, I'm unsure of the value of verifying all of these, especially when we have to deal with this slowing down CI.

I'd recommend a single platform and the maximum feature set for which our dependencies are committed to also validating this.